NewHorizons/External/Modules/PropModule.cs

```diff
@@ -98,6 +98,48 @@ public class PropModule
 
         [Obsolete("audioVolumes is deprecated. Use Volumes->audioVolumes instead.")] public VolumesModule.AudioVolumeInfo[] audioVolumes;
 
+        [JsonObject]
+        public abstract class GeneralPointPropInfo
```

Comment:
Since this is used in other modules maybe it should go in its own file, could apply the same to all the other nested Module/Info classes out there, could be cleaner